hack/validate-owners.sh

```diff
@@ -19,7 +19,7 @@ EOF
 )
 
 # mindepth for "jobs" and "configs" is 2, while "step-registry" is 1, so do these in separate steps
-no_owners_jobs_configs=$(find ci-operator/config/ ci-operator/jobs/ -mindepth 2 -type d ! -exec test -e '{}/OWNERS' \; -print | sort)
+no_owners_jobs_configs=$(find ci-operator/config/ ci-operator/jobs/ -mindepth 2 -type d !  -path ci-operator/config/openshit-priv -prune -path ci-operator/jobs/openshit-priv -prune -exec test -e '{}/OWNERS' \; -print | sort)
```

Comment:
Definitely not `openshit` :D

Reply:
Well, the whole thing is wrong. 
should be `-not -path "*openshift-priv*"`